src/app/components/osd/osd.component.ts

```diff
@@ -185,11 +190,14 @@ export class OsdComponent implements AfterViewInit, OnDestroy {
 
         this.viewer.addHandler('page', ({ page }) => {
           this.pageChange.next(page + 1);
+          this._page = page;
+          this.data.emit(page);
```

Comment:
We already have a page change event. When and why do you need a new one?

Reply:
this.data.emit (page); is used to keep track of page changes through the arrow buttons

Reply:
it is unified. The change page event should be agnostic with regards to the reason it has been changed